### PR TITLE
firebase-cli 3.6.1 (new formula)

### PR DESCRIPTION
**Pull request by cglong**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

---
`brew audit --strict firebase-cli` seems to be failing for both BrewBot and me; I'm not really sure what's causing the issue:
```
Error: undefined method `[]' for nil:NilClass
Please report this bug:
  http://docs.brew.sh/Troubleshooting.html
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:752:in `block in audit_revision_and_version_scheme'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:751:in `each'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:751:in `audit_revision_and_version_scheme'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:1239:in `block in audit'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:1232:in `each'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:1232:in `audit'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:82:in `block in audit'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:78:in `each'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:78:in `audit'
/usr/local/Homebrew/Library/Homebrew/brew.rb:90:in `<main>'
```

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
> brew audit --strict firebase-cli seems to be failing for both BrewBot and me; I'm not really sure what's causing the issue:

Fixed in https://github.com/Homebrew/brew/pull/2553

File: Formula/firebase-cli.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require "language/node"
+
+class FirebaseCli < Formula
+  desc "Firebase Command Line Tools"
```
Comment (MikeMcQuaid) [inline Formula/firebase-cli.rb]: `Description should use "command-line" instead of "Command Line"`

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

File: Formula/firebase-cli.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    # other commands require login to use
+    system bin/"firebase", "--help"
```
Comment (ilovezfs) [inline Formula/firebase-cli.rb]: Maybe something like
```
    (testpath/"login-test").write <<-EOS.undent
      spawn firebase login:ci --no-localhost
      expect "Paste" 
    EOS
    assert_match "authorization code", shell_output("expect -f login-test")
```

Comment (cglong) [inline Formula/firebase-cli.rb]: Thanks @ilovezfs; I had no idea `expect` existed 🙂 This works perfectly!

**Comment by fxcoudert** [comment]
Thank you @cglong for contributing to Homebrew!

**Pull request closed without merging**